CODEOWNERS

```diff
@@ -12,6 +12,7 @@
 src/include/sof/dmic.h			@singalsu
 src/include/host/*			@ranj063
 src/include/uapi/*			@bardliao
+src/include/uapi/**			@wwittbrx @zrombel
```

Comment:
I guess you want it to have also @bardliao in `src/include/uapi/**` otherwise line 14 has no effect

Reply:
Won't bardlio be in only for abi.h ?

Reply:
Yeah, lets have @wwittbrx, @zrombel, @bardliao and @dbaluta as UAPI reviewers. We can remove the old UAPI entry.